add-on/chrome/bootstrap.js

```diff
@@ -1039,6 +1039,14 @@ function loadDefaultPrefs() {
       }
     }
   });
+
+  if (Services.vc.compare(Services.appinfo.version, "47.0a1") < 0) {
+    branch.setBoolPref("loop.remote.autostart", false);
```

Comment:
Right, but I don't want to mess up the test audience during any possible overlap. Nor do I want testers trying to use it accidentally ;-)

Turning the pref off in GA won't actually affect anyone.